examples/misc/compare-with-kie-editors-standalone/README.md

```diff
@@ -0,0 +1,29 @@
+# Compare bpmn-visualization and kie-editors-standalone when loading Local BPMN Diagrams
+
+Javascript example
+- [__:fast_forward: live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/misc/compare-with-kie-editors-standalone/index.html)
+- to run locally, run a web server to open the [index.html](index.html) the page. See the [explanations in the repository README](../../../README.md#running-examples-locally)
+
+
+## Overview
+
+This example let you compare [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) with [kie-editors-standalone](https://github.com/kiegroup/kogito-tooling/tree/master/packages/kie-editors-standalone) on
+- BPMN elements rendering
+- API usage
+
+**WARN**
+the following applies at least to `kie-editors-standalone@0.8.1`
+- the  js bundle is very large (more than 38mb), very slow to load and generates a lot of error in the console
+- it is unable to display most of the C.x diagrams from the miwg-test-suite (parsing errors)
+- you must serve this example with a http server (see link on top of this file), otherwise, the diagram won't load in kie-editors-standalone (`window.location.origin` issue)
```

Comment:
```suggestion
- You must serve this example with a http server (see link on top of this file), otherwise, the diagram won't load in kie-editors-standalone (`window.location.origin` issue).
```